backend/src/facebook-feed/facebook-feed.service.ts

```diff
@@ -18,9 +18,7 @@ export class FacebookFeedService {
     }
 
     onModuleInit() {
-        setInterval(() => {
-            this.fetchAndSaveFacebookFeeds();
-        }, config.updateInterval);
+        this.fetchAndSaveFacebookFeeds();
```

Comment:
It's a good practice to keep related changes together and make separate PRs to fixes.

Reply:
Thank you :-) I have updated.